### DISCUSSION TITLE
Feature request: debug a subtest

### DISCUSSION THREAD

**Discussion by seancaffery** [golang/vscode-go#1432] [Discussions]
Similar to https://github.com/golang/vscode-go/pull/87 I'd like to be able to debug a specific subtest.

The current workflow requires me to skip or comment out other subtests so the regular `Debug Test At Cursor` can run specific test I want to debug.

Is there currently a way to do this? If not, I'd be happy to put together a PR.

From my limited experimentation, it looks like the implementation of #87 could be almost entirely reused.

**Comment by hyangah** [comment]
I don't find from the https://github.com/golang/vscode-go/blob/master/docs/commands.md so I assume there is no way. PR for "Debug Subtest At Cursor" is greatly appreciated. :-)

FYI I usually let the entire test run with `debug test` codelens but use conditional breakpoint on the subtest name - that works for table-driven tests as well.
